### PR TITLE
G2 2024 v5 #15191 f

### PR DESCRIPTION
**Pull request by a22iliru**
The test does create a new course and deletes it after the test is done. However when opening the file it wont show what passes and what does not unless you delete the 'query before test 'query after test' in this file. I can change if the tester has suggestion on how to fix it, otherwise i suggest merge and look how to make it shown on the page.

I also fixed test.php include_once(sessions.php) and include_once(coursepsysw.php something)

### CODE REVIEW

File: DuggaSys/tests/microservices/createNewCourse_ms_test.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                'coursecode' => 'TEST001',
+                'courseGitURL' => 'http://localhost/LenaSYS/DuggaSys/sectioned.php?courseid=9999&coursename=Test%20Course&coursevers=8888',
+                'AUtoken' => 1
+            )
+        ),
+        'filter-output' => serialize(
+            array(
+                'none' 
+            )
+        ),
```
Comment (a22albjo) [inline DuggaSys/tests/microservices/createNewCourse_ms_test.php:30]: In the future if possible filter out some of the output if there are parts that are not relevant to the service.

File: DuggaSys/tests/microservices/createNewCourse_ms_test.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ),
+);
+testHandler($testsData, true); // true = prettyprint (HTML), false = raw JSON
```
Comment (a22albjo) [inline DuggaSys/tests/microservices/createNewCourse_ms_test.php:33]: You should add a "?>" to mark the end of the file.

**Review by a22albjo — CHANGES_REQUESTED** [review]
The structure looks good, however there are a number of things that needs to be adressed.

Use the retriveCourseedService_ms.php in the microservice you are testing to get the right output. 

I have not gone in to details testing the test it self, i think it is easier to do when the comments listed bellow have been adressed.

**Comment by a22marca** [comment]
Creating a new pull request from a new branch

**Pull request closed without merging**